app/model/package.scala

```diff
@@ -63,6 +63,10 @@ trait UserMetadataLike {
   }
 }
 
+object UserMetadataLike {
+  val sortChronologically: ((String,String)) => String = (checkpoint) => checkpoint._2
```

Comment:
I'm hesitating if we shouldn't parse this timestamp.
In theory you can have eg. timestamps with different timezones.

Reply:
How would be get different timezones returned?